helm_deploy/hmpps-find-and-refer-an-intervention-service/values.yaml

```diff
@@ -1,18 +1,18 @@
 generic-service:
-  nameOverride: hmpps-template-kotlin
-  productId: "UNASSIGNED" # productId for the product that this belongs too, i.e. DPS001, see README.md for details
+  nameOverride: hmpps-find-and-refer-an-intervention-service
+  productId: "HMPPS545" # productId for the product that this belongs too, i.e. DPS001, see README.md for details
 
   replicaCount: 4
```

Comment:
Can we set replicaCount to 2 to start with ?

Reply:
Done